source/_docs/composer.md

```diff
@@ -1,175 +1,87 @@
-       "wp-content/themes/{$name}/": ["type:wordpress-theme"]
-       },
-     "dropin-paths": {
-        "wp-content": [
-        "package:wpackagist-plugin/wp-redis:object-cache.php"
-      ]
-    }
-  }
- ```
+Composer is used to fetch dependencies declared by the project as part of a CircleCI build step. This ensures that the final composed build results are installed on Pantheon:
+
+![Artifact Deployment](/source/docs/assets/images/artifact-deployment.png)
+
+### GitHub Pull Requests
+One advantage of managing code this way is that it keeps the change sets (differences) for pull requests as small as possible. If a pull request upgrades several dependencies, only the dependency metadata file will change; the actual code changes in the upgraded dependencies themselves are not shown.
+
+GitHub pull requests (PRs) are a formalized way of reviewing and merging a proposed set of changes to the source repository. When one member of a development team makes changes to a project, all of the files modified to produce the feature are committed to a separate branch, and that branch becomes the basis for the pull request. GitHub allows other team members to review all of the differences between the new files and their original versions, before merging the PR to accept changes.
+
+In this workflow, a [Multidev](/docs/multidev/) environment is created on Pantheon for each pull request branch on GitHub. Work in these environments can also be committed back to the same branch for review on GitHub. When a pull request is merged into the default branch on GitHub, the result is deployed to the Dev environment on Pantheon:
+
+![Multidev PR workflow](/source/docs/assets/images/pr-workflow/github-pr-diagram.png)
+
+### Automated Tests
+
+It is also common to set up automated tests to confirm that the project is working as expected; when tests are available, GitHub will run them and display the results of the tests with the pull request. Working on projects with comprehensive tests increases the development team's confidence that submitted pull requests will work correctly when they are integrated into the main build.
+
+### Scaling Considerations
+We recommend the "source" and "artifact" workflow for single site use cases, and for most use cases involving larger site portfolios such as EDUs. You can create a scaffold repository based off our example repositories and customize it to your liking, then use the scaffold to create new sites similar to how you would create the codebase for a [Custom Upstream](/docs/custom-upstream/).
+
+However, unlike Custom Upstreams this method does not support one-click updates in the Site Dashboard. Adopting this workflow means forgoing all other update techniques in favor of Composer. If your use case requires an simpler update strategy for non-technical site admins, this workflow could present problems scaling or at the very least require additional training for your development team.
```

Comment:
On L79 You're referencing Custom Upstreams before defining/explaining the term, which you do in the next section. I would suggest moving this down; going over custom upstreams before comparing the two methods.

Reply:
Or alternately, start by saying what you can't do, and allude that the next section will cover a method which will handle it.